source/fundamentals/gridfs.txt

```diff
@@ -10,11 +10,12 @@ GridFS
    :depth: 2
    :class: singlecol
 
-This guide shows you how to store and retrieve large files in MongoDB using
-**GridFS**. GridFS is a specification implemented by the driver that describes
-how to split files into chunks when storing them and reassemble them when
-retrieving them. The driver implementation of GridFS is an abstraction that
-manages the operations and organization of the file storage.
+In this guide, you can learn how to store and retrieve large files in
+the MongoDB Java driver using **GridFS**. GridFS is a specification
```

Comment:
```suggestion
MongoDB using **GridFS**. GridFS is a specification
```
Issue: I think this was correct before. I believe the files are stored on the MongoDB server rather than in memory in the driver.